lib/logstash/plugin_mixins/kafka/avro_schema_registry.rb

```diff
@@ -68,6 +86,23 @@ def check_for_schema_registry_connectivity_and_subjects
       if schema_registry_key and !schema_registry_key.empty?
         options[:auth] = {:user => schema_registry_key, :password => schema_registry_secret.value}
       end
+      if schema_registry_ssl_truststore_path and !schema_registry_ssl_truststore_path.empty?
+        options[:ssl] = {} unless options.key?(:ssl)
+        options[:ssl].merge!({
+          :truststore => schema_registry_ssl_truststore_path,
+          :truststore_password => schema_registry_ssl_truststore_password.value,
+          :truststore_type => schema_registry_ssl_truststore_type,
+        })
+      end
+      if schema_registry_ssl_keystore_path and !schema_registry_ssl_keystore_path.empty?
+        options[:ssl] = {} unless options.key? :ssl
+        options[:ssl].merge!({
+          :keystore => schema_registry_ssl_keystore_path,
+          :keystore_password => schema_registry_ssl_keystore_password.value,
```

Comment:
Same as `schema_registry_ssl_truststore_password `
```suggestion
          :keystore_password => schema_registry_ssl_keystore_password&.value,
```

Reply:
solved by 4914eb4eb3ddde5094fde9d960ebc697be6fad34